lib/dasher.js

```diff
@@ -31,6 +33,29 @@ function DasherButton(button) {
   doLog(button.name + " added.")
 }
 
+function doCommand(command, name, callback) {
+
+  const exec = require('child_process').exec;
+  // Assuming that the button name is trusted as it comes from the config file
+  // If it isn't true then need to do encoding here
+  exec(command + " '" + name + "'", (error, stdout, stderr) => {
```

Comment:
Building a string and executing it is fraught with security issues.

I recommend instead that you switch to `child_process.spawn`, which can take arguments as an array.